hardhat.config.ts

```diff
@@ -1,106 +1,31 @@
-// this needs to come first
 import "module-alias/register";
 
 import "@nomiclabs/hardhat-etherscan";
 import "@nomiclabs/hardhat-waffle";
 import "@openzeppelin/hardhat-upgrades";
 import "@typechain/hardhat";
-import { config as dotenvConfig } from "dotenv";
+import * as dotenv from "dotenv";
 import "hardhat-deploy";
 import "hardhat-gas-reporter";
 import { HardhatUserConfig } from "hardhat/config";
-import { NetworkUserConfig } from "hardhat/types";
-import { resolve } from "path";
 import "solidity-coverage";
 import "solidity-docgen";
 
 import "./tasks";
 
-dotenvConfig({ path: resolve(__dirname, "./.env") });
-
-// Ensure that we have all the environment variables we need.
-const privateKey: string | undefined = process.env.PRIVATE_KEY;
-if (!privateKey) {
-  throw new Error("Please set your PRIVATE_KEY in a .env file");
-}
-
-const infuraApiKey: string | undefined = process.env.INFURA_API_KEY;
-if (!infuraApiKey) {
-  throw new Error("Please set your INFURA_API_KEY in a .env file");
-}
-
-const chainIds = {
-  bsc: 56,
-  bsctestnet: 97,
-  hardhat: 31337,
-  goerli: 5,
-  rinkeby: 4,
-};
-
-function getChainConfig(chain: keyof typeof chainIds): NetworkUserConfig {
-  let jsonRpcUrl: string;
-  switch (chain) {
-    case "bsc":
-      jsonRpcUrl = process.env.BSC_RPC || "https://bsc-dataseed1.binance.org";
-      break;
-    case "bsctestnet":
-      jsonRpcUrl = process.env.BSC_TESTNET_NODE || "https://data-seed-prebsc-1-s1.binance.org:8545";
-      break;
-    default:
-      jsonRpcUrl = "https://" + chain + ".infura.io/v3/" + infuraApiKey;
-  }
-  return {
-    accounts: [`0x${privateKey}`],
-    chainId: chainIds[chain],
-    url: jsonRpcUrl,
-  };
-}
+dotenv.config();
 
 const config: HardhatUserConfig = {
-  defaultNetwork: "hardhat",
-  namedAccounts: {
-    deployer: {
-      default: 0, // here this will by default take the first account as deployer
-    },
-  },
-  etherscan: {
-    apiKey: {
```

Comment:
Don't we need this section to verify the code in bscscan?

Reply:
We need this, added back [here](https://github.com/VenusProtocol/oracle/pull/54/commits/188a7f2040e75727da892d435c0eb1f743f904de)